src/controllers/livetv/livetvsuggested.js

```diff
@@ -1,392 +1,400 @@
     }
 
-    return function (view, params) {
-        function enableFullRender() {
-            return new Date().getTime() - lastFullRender > 3e5;
-        }
+    function onTabChange(evt) {
+        const previousTabController = tabControllers[parseInt(evt.detail.previousIndex)];
 
-        function onBeforeTabChange(evt) {
-            preLoadTab(view, parseInt(evt.detail.selectedTabIndex));
+        if (previousTabController && previousTabController.onHide) {
+            previousTabController.onHide();
         }
 
-        function onTabChange(evt) {
-            var previousTabController = tabControllers[parseInt(evt.detail.previousIndex)];
+        loadTab(view, parseInt(evt.detail.selectedTabIndex));
+    }
 
-            if (previousTabController && previousTabController.onHide) {
-                previousTabController.onHide();
-            }
+    function getTabContainers() {
+        return view.querySelectorAll('.pageTabContent');
+    }
 
-            loadTab(view, parseInt(evt.detail.selectedTabIndex));
-        }
+    function initTabs() {
+        mainTabsManager.setTabs(view, currentTabIndex, getTabs, getTabContainers, onBeforeTabChange, onTabChange);
+    }
 
-        function getTabContainers() {
-            return view.querySelectorAll('.pageTabContent');
-        }
+    function getTabController(page, index, callback) {
+        let depends;
 
-        function initTabs() {
-            mainTabsManager.setTabs(view, currentTabIndex, getTabs, getTabContainers, onBeforeTabChange, onTabChange);
-        }
+        // TODO int is a little hard to read
+        switch (index) {
+            case 0:
+                depends = 'controllers/livetv/livetvsuggested';
+                break;
+
+            case 1:
+                depends = 'controllers/livetv/livetvguide';
+                break;
 
-        function getTabController(page, index, callback) {
-            var depends = [];
+            case 2:
+                depends = 'controllers/livetv/livetvchannels';
+                break;
 
-            // TODO int is a little hard to read
-            switch (index) {
-                case 0:
-                    break;
+            case 3:
+                depends = 'controllers/livetv/livetvrecordings';
+                break;
 
-                case 1:
-                    depends.push('controllers/livetv/livetvguide');
-                    break;
+            case 4:
+                depends = 'controllers/livetv/livetvschedule';
+                break;
 
-                case 2:
-                    depends.push('controllers/livetv/livetvchannels');
-                    break;
+            case 5:
+                depends = 'controllers/livetv/livetvseriestimers';
+                break;
 
-                case 3:
-                    depends.push('controllers/livetv/livetvrecordings');
-                    break;
+            case 6:
+                depends = 'scripts/searchtab';
```

Comment:
remove this `case`: `searchtab` has been removed in favor of global search, and `livetvsuggested` has only 6 tabs.

Reply:
done